### ISSUE TITLE
Error! must be string or buffer, not None

### ISSUE THREAD

**Issue by Bregor** [Crystalnix/termius-cli#8]
```
Enter your Server Auditor's username: pipopolam@gmail.com
Enter your Server Auditor's password:
Success!
Getting current keys and connections...
Success!
Decrypting keys and connections...
Error! must be string or buffer, not None
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/serverauditor_sshconfig/core/application.py", line 32, in wrapped
    func(self)
  File "/usr/local/lib/python2.7/site-packages/serverauditor_sshconfig/core/application.py", line 157, in _decrypt_sa_keys_and_connections
    self._sa_connections = p_map(decrypt_connection, self._sa_connections)
  File "/usr/local/lib/python2.7/site-packages/serverauditor_sshconfig/core/application.py", line 151, in decrypt_connection
    con['ssh_username'] = self._cryptor.decrypt(con['ssh_username'])
  File "/usr/local/lib/python2.7/site-packages/serverauditor_sshconfig/core/cryptor.py", line 47, in decrypt
    data = self.pre_decrypt_data(data)
  File "/usr/local/lib/python2.7/site-packages/serverauditor_sshconfig/core/cryptor.py", line 38, in pre_decrypt_data
    return base64.decodestring(data)
  File "/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/lib/python2.7/base64.py", line 325, in decodestring
    return binascii.a2b_base64(s)
TypeError: must be string or buffer, not None
```

**Comment by oreales** [comment]
any news on this?

**Comment by dan-perron** [comment]
👍  Frustrating.

py2.7:
```
Decrypting keys and connections...
Error! must be string or buffer, not None
Traceback (most recent call last):
  File "/Users/djperron/serverauditor/lib/python2.7/site-packages/serverauditor_sshconfig/core/application.py", line 32, in wrapped
    func(self)
  File "/Users/djperron/serverauditor/lib/python2.7/site-packages/serverauditor_sshconfig/core/application.py", line 157, in _decrypt_sa_keys_and_connections
    self._sa_connections = p_map(decrypt_connection, self._sa_connections)
  File "/Users/djperron/serverauditor/lib/python2.7/site-packages/serverauditor_sshconfig/core/application.py", line 151, in decrypt_connection
    con['ssh_username'] = self._cryptor.decrypt(con['ssh_username'])
  File "/Users/djperron/serverauditor/lib/python2.7/site-packages/serverauditor_sshconfig/core/cryptor.py", line 47, in decrypt
    data = self.pre_decrypt_data(data)
  File "/Users/djperron/serverauditor/lib/python2.7/site-packages/serverauditor_sshconfig/core/cryptor.py", line 38, in pre_decrypt_data
    return base64.decodestring(data)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/base64.py", line 321, in decodestring
    return binascii.a2b_base64(s)
TypeError: must be string or buffer, not None
```
py3:
```
Decrypting keys and connections...
Error! expected bytes-like object, not NoneType
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/base64.py", line 519, in _input_type_check
    m = memoryview(s)
TypeError: memoryview: NoneType object does not have the buffer interface

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/serverauditor_sshconfig/core/application.py", line 32, in wrapped
    func(self)
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/serverauditor_sshconfig/core/application.py", line 157, in _decrypt_sa_keys_and_connections
    self._sa_connections = p_map(decrypt_connection, self._sa_connections)
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/serverauditor_sshconfig/core/utils.py", line 35, in <lambda>
    p_map = lambda f, it: list(map(f, it))
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/serverauditor_sshconfig/core/application.py", line 151, in decrypt_connection
    con['ssh_username'] = self._cryptor.decrypt(con['ssh_username'])
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/serverauditor_sshconfig/core/cryptor.py", line 47, in decrypt
    data = self.pre_decrypt_data(data)
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/serverauditor_sshconfig/core/cryptor.py", line 38, in pre_decrypt_data
    return base64.decodestring(data)
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/base64.py", line 561, in decodestring
    return decodebytes(s)
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/base64.py", line 553, in decodebytes
    _input_type_check(s)
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/base64.py", line 522, in _input_type_check
    raise TypeError(msg) from err
TypeError: expected bytes-like object, not NoneType
```

**Comment by klukacin** [comment]
Same problem

**Comment by EvgeneOskin** [comment]
Hi, this bug is fixed in the last release – termius 1.0.2. 

Feel free to reopen this issue if you face this problem.
